drivers/spi/spi_ll_stm32.c

```diff
@@ -31,6 +31,13 @@ LOG_MODULE_REGISTER(spi_ll_stm32);
  * error flag, because STM32F1 SoCs do not support it and  STM32CUBE
  * for F1 family defines an unused LL_SPI_SR_FRE.
  */
+#ifdef CONFIG_SOC_SERIES_STM32MP1X
+
+#define LL_SPI_SR_CRCERR LL_SPI_SR_CRCE
```

Comment:
Can we avoid this as well?
You're not supposed to (re)define LL code inside zephyr

Reply:
The goal was to "alias" the CRC error and TI frame error bits (which have the same purpose but a different name on this version of SPI peripheral)

We can also do this, but this chaining a bit more ifdefs : 

```
#ifdef CONFIG_SOC_SERIES_STM32MP1X

#define SPI_STM32_ERR_MSK (LL_SPI_SR_UDR | LL_SPI_SR_CRCE | LL_SPI_SR_MODF | \
			   LL_SPI_SR_OVR | LL_SPI_SR_TIFRE)

#else

#if defined(LL_SPI_SR_UDR)
#define SPI_STM32_ERR_MSK (LL_SPI_SR_UDR | LL_SPI_SR_CRCERR | LL_SPI_SR_MODF | \
			   LL_SPI_SR_OVR | LL_SPI_SR_FRE)
#elif defined(SPI_SR_FRE)
#define SPI_STM32_ERR_MSK (LL_SPI_SR_CRCERR | LL_SPI_SR_MODF | \
			   LL_SPI_SR_OVR | LL_SPI_SR_FRE)
#else
#define SPI_STM32_ERR_MSK (LL_SPI_SR_CRCERR | LL_SPI_SR_MODF | LL_SPI_SR_OVR)
#endif
#endif /* CONFIG_SOC_SERIES_STM32MP1X */
```

Reply:
@yboutreux this should be fine